frontend/src/components/Notification.tsx

```diff
@@ -0,0 +1,43 @@
+import { Snackbar, makeStyles } from '@material-ui/core'
+import * as React from 'react';
+import Alert from '@material-ui/lab/Alert';
+
+
+type NotificationType =  {
+    isOpen: boolean;
+    type: 'error' | 'info' | 'success' | 'warning'; 
+    message: string;
+    setisOpen: React.Dispatch<React.SetStateAction<boolean>>;
+    handleClose: any
+};
+
+const useStyles = makeStyles(theme => ({
+    root:{
+        top: theme.spacing(9) //Change this depending on size of header
+    }
+}));
+
+export default function Notification(props: NotificationType) {
+    const classes = useStyles()
+    
+    React.useEffect(() => { 
+    }, [props]);
```

Comment:
Nope

Reply:
My mistake. I think I finally understand what you mean...